### ISSUE TITLE
Fails to build image/container while launching via VSCode's Remote-Containers

### ISSUE THREAD

**Issue by KEINOS** [KEINOS/TPL-PHP-HelloWorld#59]
### How to produce

1. Clone the repo and delete the `./.git/` directory
2. Initiate by `./.devcontainer/initialize_package.php FOO`
3. Re-open the container via VSCode Remote-Containers.

```log
...
[2020-11-13T11:26:24.104Z] [PID 78653] 
========================================================================
 Installer of PHP Composer for VSCode
========================================================================
- Downloading installer of composer ...
[2020-11-13T11:26:31.191Z] [PID 78653] Composer version 2.0.6 2020-11-07 11:21:17
[2020-11-13T11:26:31.440Z] [PID 78653] Checking platform settings: OK
Checking git settings: 
[2020-11-13T11:26:31.460Z] [PID 78653] OK
Checking http connectivity to packagist: 
[2020-11-13T11:26:32.641Z] [PID 78653] OK
Checking https connectivity to packagist: 
[2020-11-13T11:26:32.897Z] [PID 78653] OK
Checking github.com rate limit: 
[2020-11-13T11:26:32.969Z] [PID 78653] OK
Checking disk free space: 
[2020-11-13T11:26:32.970Z] [PID 78653] OK
Checking pubkeys: 
[2020-11-13T11:26:32.971Z] [PID 78653] Tags Public Key Fingerprint: 57815BA2 7E54DC31 7ECC7CC5 573090D0  87719BA6 8F3BB723 4E5D42D0 84A14642
Dev Public Key Fingerprint: 4AC45767 E5EC2265 2F0C1167 CBBB8A2B  0C708369 153E328C AD90147D AFE50952
OK
Checking composer version: 
[2020-11-13T11:26:33.876Z] [PID 78653] OK
Composer version: 2.0.6
[2020-11-13T11:26:33.885Z] [PID 78653] PHP version: 7.4.12
PHP binary path: /usr/local/bin/php
OpenSSL version: OpenSSL 1.1.1d  10 Sep 2019
cURL version: 7.64.0 libz 1.2.11 ssl OpenSSL/1.1.1d
zip extension: not loaded
[2020-11-13T11:26:33.906Z] [PID 78653] ✅  MOVED: composer.phar successfully moved to ENV PATH.
[2020-11-13T11:26:34.009Z] [PID 78653] Changed current directory to /root/.composer
[2020-11-13T11:26:34.442Z] [PID 78653] Using version ^3.9 for humbug/box
[2020-11-13T11:26:34.444Z] [PID 78653] ./composer.json has been created
[2020-11-13T11:26:34.478Z] [PID 78653] Running composer update humbug/box
[2020-11-13T11:26:34.482Z] [PID 78653] Loading composer repositories with package information
[2020-11-13T11:26:34.484Z] [PID 78653] Updating dependencies
[2020-11-13T11:26:36.060Z] [PID 78653] Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - humbug/php-scoper[0.13.0, ..., 0.13.9] require jetbrains/phpstorm-stubs dev-master -> found jetbrains/phpstorm-stubs[dev-master] but it does not match your minimum-stability.
    - humbug/box 3.9.0 requires humbug/php-scoper ^0.13 -> satisfiable by humbug/php-scoper[0.13.0, ..., 0.13.9].
    - Root composer.json requires humbug/box ^3.9 -> satisfiable by humbug/box[3.9.0].

[2020-11-13T11:26:36.063Z] [PID 78653] 
Installation failed, deleting ./composer.json.
[2020-11-13T11:26:36.245Z] [PID 78653] The command '/bin/sh -c sudo bash /install_composer.sh     && composer global require humbug/box     && cp -R --force --verbose /root/.composer /home/vscode/.composer     && chown --recursive vscode:vscode /home/vscode/.composer' returned a non-zero code: 2
```

**Comment by KEINOS** [comment]
There are 2 workarounds

1. Downgrade to PHP 7.3.
2. Do not use composer v2.

**Comment by KEINOS** [comment]
fixed in #63 as well
